tasks/build/build.js

```diff
@@ -228,6 +228,17 @@ outputs hash: ${outputsHash}
 build time: ${minutes}:${seconds % 60}`
 }
 
+const copyNetworks = root => {
+  const networksPath = path.join(root, `app/networks`)
+
+  // Remove the symbolic link.
+  fs.removeSync(networksPath)
+
+  // Copy the directory.
+  fs.mkdirpSync(networksPath)
```

Comment:
you may have an extra `p` —— `fs.mkdirSync` vs. `fs.mkdirpSync`

Reply:
Those are aliases.

Reply:
There is no `fs.mkdirSync`.